sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/dsl/package.scala

```diff
@@ -452,6 +453,16 @@ package object dsl {
 
       def hint(name: String, parameters: Any*): LogicalPlan =
         UnresolvedHint(name, parameters, logicalPlan)
+
+      def sample(
+          lowerBound: Double = 0.0,
+          upperBound: Double,
+          withReplacement: Boolean = false,
+          seed: Long = Utils.random.nextLong): LogicalPlan = {
```

Comment:
Why do some params above have default values?

Reply:
Just follow the `Dataset.sample` and easy to use.